grove/alpha/jordan_gradient/jordan_gradient.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import division
+
+import numpy as np
+import pyquil.quil as pq
+from pyquil.gates import X, H
+from grove.alpha.phaseestimation.phase_estimation import phase_estimation
+
+from grove.alpha.jordan_gradient.gradient_utils import binary_to_real, \
```

Comment:
The newline is unnecessary, there's enough space for `measurements_to_bf` on this line.

Reply:
joining `measurements_to_bf` makes this line 89 characters, I think pep8 specifies a maximum of 79, no?

Reply:
You're right! Sorry, my IDE was set to 100!